wasm/solana-cli/src/commands/wallet.rs

```diff
@@ -551,6 +551,12 @@ pub async fn process_airdrop(
             Ok(build_balance_message(current_balance, false, true))
         }
     } else {
+        PgTerminal::log_wasm(
+            &format!(
+                "In case of rate limit try the web faucet https://faucet.solana.com/?walletAddress={:?}&amount=5 or this guide: https://solana.com/developers/guides/getstarted/solana-token-airdrop-and-faucets",
```

Comment:
Same issue but [different solution](https://github.com/coral-xyz/anchor/blob/879601e63283065efb852ac3e991db23a7b22a7a/cli/src/checks.rs#L115-L119). `rustfmt` has max 100 characters limit by default.

Reply:
Hey. @acheroncrypto sorry, but im too stupid to format this correctly. My formatter settings seem to be different. How can I format the rust code like the CI does?